package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "chalk": "^3.0.0",
     "cosmiconfig": "^6.0.0",
-    "create-jest-runner": "^0.6.0",
+    "create-jest-runner": "^0.12.2",
```

Comment:
this is a breaking change, so we cannot do it (or, it should at least be in a separate PR)

Reply:
Still separate, yeah - fixing output bug is not the same as adding more precise errors, and it's easier to look at them separately. But that PR can manually filter the results, perhaps? Instead of bumping the dep

Reply:
Well, we can't filter the results in this codebase anywhere if we're using the `pass` function, since the bug is between that function and the underlying jest code (actually the bug is in [`toTestResult`](https://github.com/jest-community/create-jest-runner/blob/v0.6.0/lib/toTestResult.js#L35) in v0.6.0).  But what we could do is to replace the `pass` call with manually constructing the result like I'm doing above, and that would prevent having to bump the dependency.  Is that your preference?

Reply:
OK, I've added a commit to not upgrade `create-jest-runner` and to construct the results manually for `pass` (and `skip`, which I had missed before).  Also added a comment above the function explaining why we're doing it.  Let me know if this approach works now.